CLAUDE.md

```
@@ -1007,6 +1007,72 @@ obtest.AssertLogTypeExists(t, tp.LogExporter, "action")
+            metric.WithAttributes(
+                attribute.String("order_type", req.Type),
+                attribute.String("status", "success"),
+            ),
+        )
+    }
+    // ... business logic
+}
+```
+
+**Metric Types:**
+| Type | Use Case | Example |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)